CedarJavaFFI/Cargo.toml

```diff
@@ -23,3 +22,8 @@ partial-eval = ["cedar-policy/partial-eval"]
 # Dynamic linked library, used for linked to java process at runtime.
 [lib]
 crate_type = ["cdylib"]
+
+[dependencies.cedar-policy]
+version = "4.0.0"
+git = "https://github.com/cedar-policy/cedar"
+branch = "main"
```

Comment:
Instead of this block, why not the equivalent

```
cedar-policy = { version = "4.0.0", git = "https://github.com/cedar-policy/cedar", branch = "main" }
```
in `[dependencies]`?

I think we have another repo that uses this block specifically because there's some GitHub Actions hackery that involves fiddling with Cargo.toml, but are we doing that here?

Reply:
I don't think so. These hackeries were introduced because we didn't have proper version handling.

Reply:
@cdisselkoen I think the equivalent would be fine here. We can adjust in a future rev.